acceptance/acceptance_test.go

```diff
@@ -55,16 +55,18 @@ func TestAcceptance(t *testing.T) {
 	h.RequireDocker(t)
 	rand.Seed(time.Now().UTC().UnixNano())
 
+	assert := h.NewAssertionManager(t)
+
 	dockerCli, err = client.NewClientWithOpts(client.FromEnv, client.WithVersion("1.38"))
-	h.AssertNil(t, err)
+	assert.Nil(err)
```

Comment:
Visually this new `assert.X` style looks like a 🏆 .

Reply:
It feels nice to work with, I've put it in `testhelpers` too, so immediately usable in all our test suites.